spec/stashcache.spec

```diff
@@ -84,12 +110,19 @@ rm -rf %{_buildroot}
 %{python_sitelib}/xrootd_cache_stats.py*
 
 %files origin-server
-%config(noreplace) %{_sysconfdir}/xrootd/xrootd-stashcache-origin-server.cfg
-%config(noreplace) %{_sysconfdir}/xrootd/xrootd-stashcache-origin-server-itb.cfg
+%attr(-, xrootd, xrootd) %config(noreplace) %{_sysconfdir}/xrootd/xrootd-stashcache-origin-server.cfg
```

Comment:
Is this right?  I would prefer if xrootd didn't have the ability to edit its own configuration file.

Reply:
I'd followed the upstream pattern of making the config files owned by xrootd:
https://github.com/xrootd/xrootd/blob/master/packaging/rhel/xrootd.spec.in#L711

I agree it does seem unsettling in the new write-enabled xrootd world. Is that worth opening a PR with the xrootd folks to change?

Reply:
Erm - I think I'd just prefer to diverge from upstream practice here.